doc: pdn: minor updates (#6390)
nrfconnect/sdk-nrf: doc/nrf/libraries/modem/pdn.rst

The modem creates a PDN connection for a PDP context as necessary.
Multiple PDP contexts might share the same PDN connection if they are configured similarly.

Configuration
*************

> The PDN library will override the default PDP context configuration during initialization (:c:func:`pdn_init`) if the :kconfig:`CONFIG_PDN_DEFAULTS_OVERRIDE` Kconfig option is set.

Reviewer: ```suggestion
The PDN library overrides the default PDP context configuration during initialization (:c:func:`pdn_init`) if you have set the :kconfig:`CONFIG_PDN_DEFAULTS_OVERRIDE` Kconfig option.
```

The default PDP context configuration consists of the following parameters, each controlled with a Kconfig setting:

* Access point name, :kconfig:`CONFIG_PDN_DEFAULT_APN`
* Family, :kconfig:`CONFIG_PDN_DEFAULT_FAM`
* Authentication method, :kconfig:`CONFIG_PDN_DEFAULT_AUTH`
* Authentication credentials, :kconfig:`CONFIG_PDN_DEFAULT_USERNAME` and :kconfig:`CONFIG_PDN_DEFAULT_PASSWORD`

.. note::
   The default PDP context configuration must be overridden before the device is registered with the network.

Limitations
***********

> The callback for the default PDP context must be set before the device is registered to the network (``CFUN=1``) to receive the first activation event.

Reviewer: ```suggestion
You have to set the callback for the default PDP context before the device is registered to the network (``CFUN=1``) to receive the first activation event.
```
